app/subscriber/src/features/my-reports/edit/content/stories/ContentForm.tsx

```diff
@@ -34,10 +34,21 @@ export const ContentForm: React.FC<IContentFormProps> = ({
   if (!content) return null;
 
   const userId = userInfo?.id ?? 0;
-  const headline = content.versions?.[userId]?.headline
-    ? content.versions[userId].headline ?? ''
-    : content.headline;
   const isAV = content.contentType === ContentTypeName.AudioVideo;
+  const versions = content.versions?.[userId] ?? {
```

Comment:
Had to make changes here to remove the code bug, but also force the build to update this component.